### ISSUE TITLE
Text input disabled when switching tabs on safari PWA

### ISSUE THREAD

**Issue by RcrossF** [microsoft/vscode-remote-release#7167]
Operating Environment:
iPad on iOS 15.4.1
VScode remote running on windows 10 host
Connecting through safari

Steps to reproduce:
1. Connect to host
2. Open a folder and file
3. Tab out of vscode to another app
4. Tab back in
5. No text can be entered until the file is closed and reopened

**Comment by eleanorjboyd** [comment]
Hello! Could you provide a screenshot or video/gif to demonstrate the issue? This will help us determine which part of the UI you are referencing. Thanks!

**Comment by RcrossF** [comment]
Sure, here I just show tabbing back into vscode, and no text input working until I switch editor tabs. https://share.icloud.com/photos/0e8XiKxNImSCyzQ_849WWVMTQ

**Comment by RcrossF** [comment]
I should have clarified this is with code-server which is running on the windows host. I didn't realize this repo was for other remote extensions too.

**Comment by TylerLeonhardt** [comment]
@RcrossF sorry I think the video has become unaccessible for us. You should be able to attach it to the GH thread

**Comment by RcrossF** [comment]
Tested again today and the issue is still present in 1.74.1. Sorry for the delay.

https://user-images.githubusercontent.com/16991582/207946246-9ffd5f78-df99-45a2-bc90-d0bf0625235d.MOV

**Comment by TylerLeonhardt** [comment]
In your video, when you first move over to VS Code, you are unable to type in the editor is that right?

Can you try when you have your cursor in a different part of VS Code UI? Like maybe the search view box?

**Comment by RcrossF** [comment]
Yeah, switching back to vs code and tapping in the code editor doesn't let me type. Search is unaffected, seems to just be the editor. Switching the cursor from code -> search -> code or switching code tabs fixes it until I switch tasks again.
